compiler/optimizations/gpuTransforms.cpp

```diff
@@ -572,16 +572,23 @@ bool GpuizableLoop::parentFnAllowsGpuization() {
   return true;
 }
 
-// forall loops that contain a reduction intent introduce a temporary variable
-// with a special flag that we'll look for (for the time being we want to
-// not gpuize these loops).
-bool GpuizableLoop::containsReductionTemporary() {
+
+bool GpuizableLoop::containsNonGpuizablePattern() {
```

Comment:
The name is not as self-documenting as I'd like. Consider `symsInBodyAreGpuizable`. It doesn't describe what we just did well, but I like better than the very general function name you have.